Code/Framework/AzCore/AzCore/Component/Component.h

```diff
@@ -240,9 +240,8 @@ namespace AZ
          */
         virtual void SetEntity(Entity* entity);
 
-        //! Function to call before adding this component to the provided entity.
-        //! @param entity The entity to add the component to.
-        virtual void OnPrepareForAdditionToEntity(Entity* entity);
+        //! Function to call after setting the entity in this component.
+        virtual void OnAfterEntitySet();
```

Comment:
looks like the API change wasn't included in the last release so this change should be OK - otherwise you'd want to  follow the API deprecation guidelines https://github.com/o3de/community/blob/main/guides/o3de-deprecation-guidelines.md

Reply:
Oh, I didn't notice that. thanks for checking this!